src/frontend/style/main.css

```diff
@@ -1,3 +1,6 @@
+/* Font Used for Search Icon */
+@import url('https://fonts.googleapis.com/css?family=Inconsolata:700');
```

Comment:
Maybe keep font dependencies to a minimum? Not only for a more consistent design but also to avoid loading a bunch (although this one is only ~60KB, but it adds up).

Reply:
changed it to the main font we use in HTML.